src/wasm-v8.cc

```diff
@@ -532,7 +487,7 @@ auto Store::make(Engine*) -> own<Store> {
   store->context()->Enter();
   isolate->SetData(0, store.get());
 
-  return make_own(seal<Store>(store.release()));
+  return make_own<Store>(store.release());
```

Comment:
```suggestion
  return own<Store>(store.release());
```

Reply:
I've removed make_own entirely. This line no longer needs a cast at all.